### ISSUE TITLE
WorkspaceConfigurationService.lookup does not contain value 'null'

### ISSUE THREAD

**Issue by aeschli** [Microsoft/vscode#21282]
'null' is a valid value for a setting, but WorkspaceConfigurationService.lookup turns it into 'undefined'.

- have the following workspace settings:
```json
{
   "workbench.iconTheme": null
}
```
- set a breakpoint in 'themes.contribution.js' at `this.configurationService.lookup(themeService_1.ICON_THEME_SETTING)`
- the value is
```javascript
{
   ...
   workspace: undefined
}
```

'undefined' should indicate that the setting has not been set in the workspace (luckily it is not a valid configuration value as we only support valid JSON data types)

**Comment by aeschli** [comment]
Same problem with the default and user value. It's always 'undefined' instead of 'null'.
The bug is in https://github.com/Microsoft/vscode/blob/master/src/vs/platform/configuration/common/configuration.ts#L91

**Comment by aeschli** [comment]
I created a PR with a fix: #21289
